lib/resty/etcd/v3.lua

```diff
@@ -593,6 +593,7 @@ local function http_request_chunk(self, http_cli)
         ssl_verify = self.ssl_verify,
         ssl_cert_path = self.ssl_cert_path,
         ssl_key_path = self.ssl_key_path,
+        ssl_server_name = self.sni,
```

Comment:
Can we add a test to cover http_request_chunk?

Reply:
update test case to cover it.

additional notes:

1. in request_uri, if there is no sni, will use host as default sni, see: https://github.com/api7/lua-resty-http/blob/master/lib/resty/http.lua#L916
2. in request_chunk, if there is no sni, it is nil.

And if sni is nil, the sni and certificate host will not be verified.